plugins/inputs/snmp/snmp.go

```diff
@@ -24,9 +24,15 @@ import (
 
 const description = `Retrieves SNMP values from remote agents`
 const sampleConfig = `
+  [[inputs.snmp]]
```

Comment:
I think this line addition is causing the test suite to fail..

Reply:
I removed the line.
Thank you for your patience!